templates/registration/profile.html

```diff
@@ -6,34 +6,33 @@
 {% endblock %}
 
 {% block content %}
-    <h1>Profile</h1>
-    <h2 class="profile-header">Welcome, {% firstof user.get_full_name user.get_username %}!</h2>
+    <h2 class="profile-header">{% firstof user.get_full_name user.get_username %}'s Profile</h2>
```

Comment:
Heads up: there's a gotcha with using possessive on someone's name. For names that end in 's', this looks weird. ie. "Lucas's Profile"